core/src/test/java/org/elasticsearch/common/settings/ScopedSettingsTests.java

```diff
@@ -319,7 +316,7 @@ public void testLoggingUpdates() {
             settings.applySettings(Settings.builder().put("logger._root", "TRACE").build());
             assertEquals("TRACE", ESLoggerFactory.getRootLogger().getLevel());
             settings.applySettings(Settings.builder().build());
-            assertEquals(level, ESLoggerFactory.getRootLogger().getLevel());
+            assertEquals(property != null ? property : level, ESLoggerFactory.getRootLogger().getLevel());
```

Comment:
`property` can never be `null` in the updated code (though if you wanted to add a `randomBoolean` to allow it, I wouldn't be opposed).